File: LorePdks.COMMON/DTO/FirmaKisi/FirmaKisiDTO.cs
using LorePdks.COMMON.DTO.Base;
using LorePdks.COMMON.DTO.Common;
using LorePdks.COMMON.DTO.Firma;
using LorePdks.COMMON.DTO.Kisi;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LorePdks.COMMON.DTO.FirmaKisi
{


    public class FirmaKisiDTO : BaseDTO
    {
        public FirmaDTO firmaDto { get; set; }
        public KisiDTO kisiDto { get; set; }
        public KodDTO firmaKisiTipKodDto { get; set; }
    }
}

File: LorePdks.WEB/src/app/core/models/FirmaKisiDto.ts
import { baseDto } from "./baseDto";
import { FirmaDto } from "./FirmaDto";
import { KisiDto } from "./KisiDto";
import { KodDto } from "./KodDto";

/**
 * Firma kişi bilgisini temsil eden model
 */
export class FirmaKisiDto extends baseDto {
  firmaDto: FirmaDto;
  kisiDto: KisiDto;
  firmaKisiTipKodDto: KodDto;
  firma_kisi_cihaz_kod?: string;
}